### PR TITLE
Set up  esbuild

### PR DESCRIPTION
**Pull request by TraceyOnim**

### CODE REVIEW

**Comment by ericmj** [comment]
I changed the base branch to `redesign` since we don't want these changes in the `main` branch while the work is in progress.

File: mix.exs [hunk 1]
```diff
@@ -69,7 +69,8 @@ defmodule Hexpm.MixProject do
       {:rollbax, "~> 0.5"},
       {:sweet_xml, "~> 0.5"},
       {:telemetry_poller, "~> 1.0"},
-      {:telemetry_metrics, "~> 0.6"}
+      {:telemetry_metrics, "~> 0.6"},
+      {:esbuild, "~> 0.2", only: :dev}
```
Comment (ericmj) [inline mix.exs:73]: Dependencies should be sorted alphabetically.

Comment (ericmj) [reply mix.exs:73]: It should be `{:esbuild, "~> 0.2", runtime: Mix.env() == :dev}` so that we can run esbuild when building for production also.

**Review by ericmj — COMMENTED** [review]
It looks good great overall. I pushed a commit to your branch with some minor fixes and removal of files that are not needed anymore.

The images were .gitignored before so they were not included in your changes but I fixed the .gitignore in my commit. If you `git pull` my commit and then do another commit they should be added to `priv/`.

File: assets/css/app.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+/* This file is for your main application css. */
```
[the remark this answers is not in the document]
Comment (ericmj) [reply assets/css/app.css:1]: This doesn't apply for none SCSS files. I have removed the SCSS linter.

File: mix.exs (hunk 1 above)
Comment (TraceyOnim) [reply mix.exs:73]: Makes sense, Working on it

File: assets/css/app.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (TraceyOnim) [reply assets/css/app.css:1]: That was confusing, thank you for the clarification there

**Comment by ericmj** [comment]
I removed some NPM deps that we no longer need.

**Review by ericmj — APPROVED** [review]

**Comment by ericmj** [comment]
Thank you! 💜

**Pull request merged**